src/MICore/LaunchOptions.cs

```diff
@@ -310,54 +341,85 @@ public bool IsValidMiDebuggerPath()
             return File.Exists(MIDebuggerPath);
         }
 
-        static internal LocalLaunchOptions CreateFromXml(Xml.LaunchOptions.LocalLaunchOptions source)
+        static internal LocalLaunchOptions CreateFromJson(string json)
         {
```

Comment:
>{ [](start = 8, length = 1)

You don't need to do it now (though it might make it easier to test, so maybe you want to), but I would suggest adding some unit tests for this stuff. It has been helpful in the XML case.